### PR TITLE
Last fix maybe

### PR DESCRIPTION
**Pull request by JonasJallet**
- fix error message back and front
- fix size and details about formular
- refactor

### CODE REVIEW

File: web-app/src/components/forms/create/CreateUserForm.tsx [hunk 1]
```diff
@@ -146,14 +143,18 @@ export default function CreateUserForm({
               id="password"
               placeholder="Mots de passe"
```
Comment (PaulineHavez) [inline web-app/src/components/forms/create/CreateUserForm.tsx:144]: Mot de passe ?

File: web-app/src/components/forms/edit/EditCategoryForm.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+  const handleFormSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    try {
+      const { data } = await updateCategoryMutation({
+        variables: updatedCategory,
+      });
+      if (data) {
+        setIsEditionPanelAdmin(false);
+        toast.success("La catégorie a bien été modifié !");
```
Comment (PaulineHavez) [inline web-app/src/components/forms/edit/EditCategoryForm.tsx:35]: modifiée*

File: web-app/src/components/forms/edit/EditUserForm.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  const handleFormSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    try {
+      const { data } = await updateUserMutation({
+        variables: updatedUser,
+      });
+      if (data) {
+        setIsEditionPanelAdmin(false);
+        toast.success("La catégorie a bien été modifié !");
```
Comment (PaulineHavez) [inline web-app/src/components/forms/edit/EditUserForm.tsx:47]: modifiée*

**Review by JohannaStoicanescu — APPROVED** [review]

**Review by PaulineHavez — APPROVED** [review]
Super nice job

**Pull request merged**